components/eamxx/CMakeLists.txt

```diff
@@ -212,8 +225,8 @@ endif()
 #     #cmakedefine RRTMGP_EXPENSIVE_CHECKS
 option (SCREAM_RRTMGP_DEBUG "Turn on extra debug checks in RRTMGP" ${SCREAM_DEBUG})
 
-option(SCREAM_RRTMGP_ENABLE_YAKL "Use YAKL under rrtmgp" TRUE)
-option(SCREAM_RRTMGP_ENABLE_KOKKOS "Use Kokkos under rrtmgp" FALSE)
+option(SCREAM_RRTMGP_ENABLE_YAKL "Use YAKL under rrtmgp" FALSE)
```

Comment:
This might be an important change. I'm checking whether reverting this fixes Frontier. FYI, @tcclevenger @jgfouca

Reply:
Yes, this must be the issue. I have a branch coming where Kokkos will be fully supported and have good performance.

Reply:
No luck. Still get the same issue. Will try something else.

Reply:
[Disable YAKL cmake stuff if we aren't using it](https://github.com/E3SM-Project/E3SM/pull/6879/commits/df5ae9082e332eda6b66f587bc0538df435bfcda) from  https://github.com/E3SM-Project/E3SM/pull/6879/commits looks promising.

Reply:
@jgfouca @tcclevenger the above commit does the trick. I got an ne30pg2 ERS test to run and pass with this and a commit (pushed to this branch) to fix a line of F90.

Reply:
@bartgol here is where the Kokkos version is being turned on.